pom.xml

```diff
@@ -25,7 +25,7 @@
         <maven.compiler.target>${jdkVersion}</maven.compiler.target>
 
         <!-- Versioning -->
-        <version.number>1.12.3</version.number>
+        <version.number>1.12.4</version.number>
```

Comment:
Leave it, it's alright. I normally bump the version only when I know that I won't commit something anymore.